.github/workflows/release-dashpay-testflight.yml

```
@@ -0,0 +1,523 @@
+    name: Build and upload dashpay
+    runs-on: macos-15
+    timeout-minutes: 180
+    environment: testflight
+    defaults:
+      run:
+        shell: bash
+        working-directory: ${{ github.workspace }}/dashwallet-ios
+
+    env:
+      APPLE_TEAM_ID: 44RJ69WHFF
+      ARCHIVE_PATH: ${{ runner.temp }}/dashpay.xcarchive
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)